src/js/tech/html5.js

```diff
@@ -282,6 +342,63 @@ class Html5 extends Tech {
     this.textTracks().removeTrack_(e.track);
   }
 
+  proxyNativeVideoTracks_() {
+    let vt = this.el().videoTracks;
+
+    if (vt && vt.addEventListener) {
+      vt.addEventListener('change', this.handleVideoTrackChange_);
+      vt.addEventListener('addtrack', this.handleVideoTrackAdd_);
+      vt.addEventListener('removetrack', this.handleVideoTrackRemove_);
+    }
+  }
+
+  handleVideoTrackChange(e) {
+    let vt = this.videoTracks();
+    this.videoTracks().trigger({
+      type: 'change',
+      target: vt,
+      currentTarget: vt,
+      srcElement: vt
+    });
+  }
+
+  handleVideoTrackAdd(e) {
+    let track = e.track;
+    if(!(e.track instanceof VideoTrack)) {
```

Comment:
Huh, weird. Also, they don't actually emit any events so they don't technically need to inherit from EventTarget: https://html.spec.whatwg.org/multipage/embedded-content.html#audiotrack.
I wonder if we can come up with a separate mechanism letting the corresponding TrackList know that the selected/enabled options changed.